packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -720,6 +774,27 @@ export class StatsAnalyzer extends EventsScope {
               this.statsResults.resolutions['video-share-send'].send.framesSent
             );
           }
+
+          // Share video is encoded but frames are not sent
+          if (
+            currentStats.framesEncoded !== previousStats.framesEncoded &&
+            (currentStats.framesSent === previousStats.framesSent || currentStats.framesSent === 0)
```

Comment:
same as above

Reply:
Changed to `this.statsResults.resolutions['video-send'].send.framesSent` and `this.lastStatsResults.resolutions['video-send'].send.framesSent`